resolwe/flow/engines/__init__.py

```diff
@@ -79,25 +83,8 @@ def communicate(self, run_sync=False, verbosity=1):
                         data.save()
                         continue
 
-                    script_template = data.process.run.get('script', '')
-                    inputs = data.input.copy()
-                    hydrate_input_references(inputs, data.process.input_schema)
-                    # hydrate_input_uploads(inputs, data.process.input_schema)
-
-                    info = {}
-                    # info['case_ids'] = data.case_ids
-                    info['data_id'] = data.id
-                    # info['data_path'] = settings.RUNTIME['data_path']
-                    # info['slugs_path'] = settings.RUNTIME['slugs_path']
-                    inputs['proc'] = info  # add script info
-
-                    try:
-                        script = template.Template('{% load resource_filters %}{% load mathfilters %}' +
-                                                   script_template).render(template.Context(inputs))
-                    except template.TemplateSyntaxError as ex:
-                        data.status = Data.STATUS_ERROR
-                        data.process_error.append('Error in process script: {}'.format(ex))
-                        data.save()
+                    script = self.exprengines['dtlbash'].eval(data)
```

Comment:
> I do not think we need FLOW_EXPRESSION_ENGINES in settings at all? We should just load all available expression engines. Because dtlbash is available and because it is hardcoded, an exception can not happen here.

But currently, the expression engines are loaded based on the `FLOW_EXPRESSION_ENGINES` setting, so an exception is possible. See the above code which does:
```python
self.exprengines = self.load_exprengines(settings.FLOW_EXPRESSION_ENGINES)
```